src/datepicker/datepicker.stories.ts

```diff
@@ -21,25 +20,27 @@ storiesOf("DatePicker", module)
 		<app-experimental-component></app-experimental-component>
 		<ibm-date-picker
 			label="Date Picker Label"
-			(selectDates)="selectDates($event)">
+			[date]="date"
+			(datesChange)="datesChange($event)">
```

Comment:
For completeness...
```suggestion
			(dateChange)="datesChange($event)">
```

You'll want to update the props and other stories as well